doc/refman/RefMan-pre.tex

```diff
@@ -1335,6 +1335,157 @@ \section*{Credits: version 8.7}
+  ``Require'' a module, by Pierre-Marie Pédrot.
+\item Universes: many user-level and API level enhancements: qualified
+  naming and printing, variance annotations for cumulative inductive
+  types, more general constraints and enhancements of the minimization
+  heuristics, interaction with modules by Gaëtan Gilbert, Pierre-Marie
+  Pédrot and Matthieu Sozeau.
+\item Decimal Numbers library by Pierre Letouzey (and DDR?)
+\item Documentation: a large community effort resulted in the migration
+  of the reference manual to the Sphynx documentation tool. The
+  result can be seen at ???
+\item Tools: experimental ``-mangle-names'' option to coqtop/coqc for
+  linting proof scripts, by Jasper Hugunin.
+\end{itemize}
+
+On the implementation side, the \texttt{dev/doc/changes.md} file
+documents the numerous changes to the implementation and improvements of
+interfaces. The file provides guidelines on porting a plugin to the new
+version.
+
+Version 8.8 also comes with a bunch of smaller-scale changes and
+improvements regarding the different components of the system. They are
+documented in the \texttt{CHANGES} file.
+
+The efficiency of the whole system has seen improvements thanks to
+contributions from Gaëtan Gilbert, Pierre-Marie Pédrot, Maxime Dénès and
+Matthieu Sozeau and performance issue tracking by Jason Gross and Paul
+Steckler.
+
+The official wiki and the bugtracker of {\Coq} migrated to the
+github platform, thanks to the work of Pierre
+Letouzey and Théo Zimmermann. Gaëtan Gilbert, Emilio Jesús Gallego Arias
+and ... worked on maintaining and improving the continuous integration
+system.
+
+The OPAM repository for {\Coq} packages has been maintained by Guillaume
+Melquiond, Matthieu Sozeau, Enrico Tassi with contributions from many
+users. A list of packages is available at \url{https://coq.inria.fr/opam/www/}.
+
+TODO Outdated?
+Packaging tools and software development kits were prepared by Michael
+Soegtrop with the help of Maxime Dénès and Enrico Tassi for Windows, and
+Maxime Dénès for MacOS X. Packages are regularly built on the
+Travis and GitLab continuous integration server.
+
+The 40 contributors for this version are Yves Bertot, Joachim
+Breitner, Tej Chajed, Arthur Charguéraud, Jacques-Pascal Deplaix, Maxime
+Dénès, Jim Fehrle, Yannick Forster, Gaëtan Gilbert, Jason Gross, Samuel
+Gruetter, Thomas Hebb, Hugo Herbelin, Jasper Hugunin, Emilio Jesus
+Gallego Arias, Ralf Jung, Johannes Kloos, Matej Košík, Robbert Krebbers,
+Tony Beta Lambda, Vincent Laporte, Pierre Letouzey, Farzon Lotfi,
+Cyprien Mangin, Guillaume Melquiond, Raphaël Monat, Carl Patenaude
+Poulin, Pierre-Marie Pédrot, Matthew Ryan, Matt Quinn, Sigurd Schneider,
+Bernhard Schommer, Matthieu Sozeau, Arnaud Spiwack, Paul Steckler,
+Enrico Tassi, Anton Trunov, Martin Vassor, Vadim Zaliva and Théo
+Zimmermann.
+
+The development process was coordinated by Matthieu Sozeau. Maxime Dénès
+was in charge of the release process. The maintainer of this release
```

Comment:
I think you should remove the sentence about the maintainer until we figure this out.

Reply:
Done